presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardManager.java

```diff
@@ -30,7 +30,7 @@
     /**
      * Create a table.
      */
-    void createTable(long tableId, List<ColumnInfo> columns, boolean bucketed, OptionalLong temporalColumnId);
+    void createTable(long tableId, boolean tableSupportsDeltaDelete, List<ColumnInfo> columns, boolean bucketed, OptionalLong temporalColumnId);
```

Comment:
Having a flag between `tableId` and `columns` is weird. Let's put it to the end.